### PR TITLE
add: debug log support

### PR DESCRIPTION
**Pull request by VladVega**
Adding debug log support.

One question is: should there be tests associated with this? I didn't find anything similar (there is successful call to runJob tested) so I thought I'd ask.

Notice the conflict: 
It's weird because the tests pass still. I noticed that when I make these same changes (happens without conflict) to master instead of `v0.6.0` there is an error when running the `juttle-engine-client`

### CODE REVIEW

**Comment by mnibecker** [comment]
There's some test for http-api in `test/utils/http-api.spec.js`. But yeah I don't think we test to verify that the correct parameters are posted. I would add a test (using nock) that verify that the addDebugLogs gets posted in the request.

File: src/utils/http-api.js [hunk 1]
```diff
@@ -41,17 +41,21 @@ export function getInputs (url, bundle, inputs = {}) {
...
 
-export function runJob(url, bundle, inputs = {}) {
+export function runJob(url, bundle, inputs = {}, addDebugLogs = false) {
+    
+    var body = {
+        bundle,
+        inputs: JSDP.serialize(inputs, { toObject: true }),
+        debug: addDebugLogs
+    };
+
```
Comment (mnibecker) [inline src/utils/http-api.js]: I would say either keep body the makeRequest statement or do the same thing in `getInputs` (for consistency sake).

**Comment by VladVega** [comment]
take another look @mnibecker ?

The error on test failure is now very clear.

**Comment by mnibecker** [comment]
+1

**Pull request merged**